rfcs/text/0041/user.yml

```
@@ -0,0 +1,199 @@
+- name: user.profile.id
+    level: extended
+    type:  keyword
+    example:  1234
+    description:  User ID from the identity datasource.
+
+- name: user.profile.type
+    level: extended
+    type:  keyword
+    example:  Employee
+    description:  Type of user account.
+
+- name: user.profile.status
+    level: extended
+    type:  keyword 
+    example: On board
+    description:  Status of the user account.
+
+- name: user.profile.first_name
+    level: extended
+    type:  keyword 
+    example: First
+    description:  First Name of the User.
+
+- name: user.profile.last_name
+    level: extended
+    type:  keyword 
+    example: Last
+    description:  Last Name of the user.
+
+- name: user.profile.other_identities
+    level: extended
+    type:  keyword 
+    multi_fields:
+        - type: text 
+    example: first.last@elk.elastic.co
+    description:  Array of additional user identities (usually email addresses).
+
+- name: user.profile.manager
+    level: extended
+    type:  keyword 
+    example: John Doe
+    description:  Assigned Manager for the user account.
+
+- name: user.profile.employee_type
+    level: extended
+    type:  keyword 
+    example: Regular 
+    description:  Further classification type for the user account.
+
+- name: user.profile.job_family
+    level: extended
+    type:  keyword 
+    example: 65-Sales
+    description:  Job family associated with the user account.
+
+- name: user.profile.job_family_group
+    level: extended
+    type:  keyword 
+    example: GTM
+    description:  Job family group associated with the user account.
+
+- name: user.profile.management_level
+    level: extended
+    type:  keyword 
+    example: Individual Contributor
+    description:  If the user account is identified as a Manager or Individual contributor.
+
+- name: user.profile.job_title
+    level: extended
+    type:  keyword 
+    example: Field Sales
+    description:  Job title assigned to the user account.
+
+- name: user.profile.department
+    level: extended
+    type:  keyword 
+    example: x256
+    description:  Department name associated with the user account.
+
+- name: user.profile.organization
+    level: extended
+    type:  keyword 
+    example: Elasticsearch Inc.
+    description:  Organization name associated with the account.
+
+- name: user.profile.location
+    level: extended
+    type:  keyword 
+    example: US - Washington - Distributed
+    description:  Assigned location for the user account.
+
+- name: user.profile.mobile_phone
+    level: extended
+    type:  keyword 
+    example: 222-222-2222
+    description: 
+
+- name: user.profile.primaryPhone
+    level: extended
+    type:  keyword 
+    example: 222-222-2222
+    description: 
+
+- name: user.profile.secondEmail
+    level: extended
+    type:  keyword 
+    example: first.l@elastic.co
+    description:  Additional email addresses associated with the user account.
+
+- name: user.profile.sup_org_id
```

```suggestion
- name: user.profile.supervisory_org_id
```

nit: slightly longer naming, but I think it makes sense to detail out `supervisory`. Also better matches the `.supervisory_org` field which uses the full word.